prover/src/aggregator/prover.rs

```diff
@@ -37,7 +41,8 @@ impl<'params> Prover<'params> {
         env::set_var("KECCAK_ROWS", BATCH_KECCAK_ROW.to_string());
 
         let prover_impl = common::Prover::from_params_map(params_map);
-        let chunk_protocol = force_to_read(assets_dir, &CHUNK_PROTOCOL_FILENAME);
+        let halo2_protocol = force_to_read(assets_dir, &FD_HALO2_CHUNK_PROTOCOL);
+        let sp1_protocol = force_to_read(assets_dir, &FD_SP1_CHUNK_PROTOCOL);
```

Comment:
so we usually generate this file inside sp1-halo2-wrapper repo, and copy it here?

Reply:
Yes, we load the SNARK `protocol` from the halo2-backend for sp1. Since that won't change post-e2e tests, this will also be published as a part of the assets for that release